gen/definitions/evpn_profile.yaml

```diff
@@ -0,0 +1,15 @@
+---
+name: EVPN Profile
+path: Cisco-IOS-XE-native:native/l2vpn/Cisco-IOS-XE-l2vpn:evpn_cont/l2-profile/evpn/profile/profile-name-list[name=%v]
+doc_category: EVPN
+test_tags: [IOSXE1715]
+attributes:
+  - yang_name: name
+    id: true
+    example: MY_EVPN_PROFILE
+  - yang_name: evi-base
+    tf_name: evi_base
```

Comment:
The `tf_name` here is redundant as by default (if not explicitly specified) it will be derived from `yang_name` and converted to snake case.